### DISCUSSION TITLE
Norfair for python 3.6.9

### DISCUSSION THREAD

**Discussion by dinarkino** [tryolabs/norfair#64] [Q&A] [answered]
Are there any critical updates of a tracker from version 0.1.8? I'm trying to use Norfair on python 3.6.9. It seems that 0.1.8 is the last version that could work on such a python version. I tried to build 0.3.0 from sources but there are requirements for python >= 3.7 for numpy and scipy, for example.

**Comment by joaqo** [comment, marked as answer]
Hi @dinarkino the only big new feature since 0.1.8 has been the addition of the ability to select your own custom kalman filter, which shouldn't be something most users need. I am using version 0.1.8 in several production projects with no issues. Please see here for more detail:

https://github.com/tryolabs/norfair/releases

**Reply by dinarkino** [reply]
Thank you!
